sycl/include/sycl/ext/intel/experimental/esimd/memory.hpp

```diff
@@ -2704,7 +2704,7 @@ __ESIMD_API std::enable_if_t<
     __ESIMD_DNS::get_num_args<__ESIMD_DNS::to_lsc_atomic_op<Op>()>() == 0,
     __ESIMD_NS::simd<T, N>>
 lsc_atomic_update(T *p, __ESIMD_NS::simd_view<Toffset, RegionTy> offsets,
-                  __ESIMD_NS::simd_mask<N> pred = 1) {
+                  __ESIMD_NS::simd_mask<N> pred) {
```

Comment:
Gregory, why do you change user-visible API so easily/bravely? Such change definitely can cause compilation fails on existing code!

For example, such code compile correctly now (without your fix):
```
            // B is USM pointer 'int *'
            simd<uint32_t, 16> Offsets(0, 8);
            uint32_t OneOffset = 4;
            //lsc_atomic_update<atomic_op::inc, int>(B, Offsets); // Doesn't work only because of inconsistency - lsc_atomic_update accepting simd Offset doesn't have default value for Mask.
            lsc_atomic_update<atomic_op::inc, int, 16>(B, OneOffset); // Compiles correctly.
            lsc_atomic_update<atomic_op::inc, int, 16>(B, Offsets.bit_cast_view<uint32_t>()); // Compiles correctly.
```